src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -63,8 +63,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String password = request.getParameter("password");
     String hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt());
+    String aboutMe = request.getParameter("aboutMe");
 
-    User user = new User(UUID.randomUUID(), username, hashedPassword, Instant.now());
+    User user = new User(UUID.randomUUID(), username, hashedPassword, Instant.now(), aboutMe);
```

Comment:
Just want to check that the corresponding test still passes even though we didn't need to update it.